sale_commission/wizard/wizard_settle.py

```diff
@@ -12,6 +12,8 @@ class SaleCommissionMakeSettle(models.TransientModel):
     _description = "Wizard for settling commissions in invoices"
 
     date_to = fields.Date("Up to", required=True, default=fields.Date.today())
+    date_payment_to = fields.Date("Date Payment Up to", default=fields.Date.today())
```

Comment:
The name of the field and the label is not very intuitive. Also non having help doesn't help, hehe. Can you please look for other name and put a help for this new field.

Reply:
@pedrobaeza since this is becoming a feature of base module, I would suggest changing labels of both date fields in the wizard for consistency: 

Invoice date up to
Payment date up to

shall we proceed?

Reply:
OK

Reply:
https://github.com/OCA/commission/pull/441/files#diff-7dfa7dd0fc23295fc2cd2da1c87e3535d33ddbf4480da661bdab6a3eaab7ea51R14

Reply:
@pedrobaeza go?